client/src/composables/useData.js

```diff
@@ -85,15 +86,14 @@ function createInstance(apiEndpoint, apiExtension) {
         data.value = res.data.rows;
         numberOfTotalEntries.value = res.data.count;
         calcRanges(offset);
-        return;
+      } else {
+        data.value = res.data;
       }
-
-      data.value = res.data;
-      errorMessage.value = "";
+      errorMessage.value = null;
     } catch (error) {
       console.error(error);
-      errorMessage.value =
-        "Beim laden der Daten ist ein Fehler aufgetreten. Bitte lade die Seite erneut.";
+      // errorMessage.value =
```

Comment:
In FlightsAll/ResultsView wird die ErrorMessage noch referenziert. Aber hier kann sie ja nur noch "null" sein.

Ich habe glaube ich noch nicht verstanden, worauf du hier hinaus willst.

Reply:
stimmt, da hab ich sie jetzt auskommentiert. ich wollte das error handling aus dem composable raushaben, da es sonst zu vermischt war denn die parent Komponenten habben ja ihr eigenes error handling. ich habe es jetzt mal auskommentiert. ich habe jetzt keine negativen Nebeneffekte gefunden. kannst ja auch mal probieren. ich hab mal den server ausgeschaltet und Seiten oder filter aufgerufen, oder die db.
ansonsten würde ich sagen machen wir in useData ein hasError prop was man true oder false setzt statt die errorMessage.

Reply:
Check. Klingt gut.